src/Extensions/ComposerUpdateExtension.php

```diff
@@ -21,21 +20,19 @@ class ComposerUpdateExtension extends DataExtension
 
     private static $db = [
         'VersionHash' => 'Varchar',
-        'LatestVersion' => 'Varchar',
-    ];
-
-    private static $summary_fields = [
-        'LatestVersion',
+        'VersionConstraint' => 'Varchar(50)',
+        'AvailableVersion' => 'Varchar(50)',
+        'AvailableHash' => 'Varchar(50)',
+        'LatestVersion' => 'Varchar(50)',
+        'LatestHash' => 'Varchar(50)',
```

Comment:
Defining some of these in a `private static $summary_fields` would be good, as a selection you wish to expose to CSV exported reports.